cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/batch/BatchJoinerRuntimeContext.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.etl.api.batch;
+
+import co.cask.cdap.api.data.schema.Schema;
+
+import java.util.Map;
+
+/**
+ * Runtime context for batch joiner
+ */
+public interface BatchJoinerRuntimeContext extends BatchRuntimeContext {
+
+  /**
+   * Retruns list of schemas for batch joiner
+   * @return
+   */
+  Map<String, Schema> getInputSchemas();
+
+  /**
+   * Returns outpuSchema for batch joiner
```

Comment:
should mention that this will return whatever was set as the output schema at configure time.